lambdas/check-existing-identity/src/test/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandlerTest.java

```diff
@@ -702,12 +707,18 @@ void shouldNoMatchStrongestVotAndAlsoVCsDoNotCorrelate() throws Exception {
     @ParameterizedTest
     @MethodSource("votAndVtrCombinationsThatShouldStartIpvJourney")
     void shouldReturnJourneyIpvGpg45MediumResponseIfNoProfileAttainsVot(
-            Map<String, Object> votAndVtr) {
+            List<String> vtr, Optional<Vot> vot) throws Exception {
         when(ipvSessionService.getIpvSession(TEST_SESSION_ID)).thenReturn(ipvSessionItem);
         when(criResponseService.getFaceToFaceRequest(TEST_USER_ID)).thenReturn(null);
+
+        List<VerifiableCredential> credentials = new ArrayList<>();
```

Comment:
This section seems to have been accidentally deleted in the move to the VC class